examples/nlp/text_classification/conf/config.yaml

```diff
@@ -0,0 +1,81 @@
+# Sentence classification with pretrained BERT models
+
+io:
+  work_dir: ./outputs
+  checkpoint_dir: null
+  save_epoch_freq: 1
+  save_step_freq: -1
+
+pl:
+  trainer:
+    gpus: 1 # the number of gpus, 0 for CPU
+    num_nodes: 1
+    max_epochs: 10
+    max_steps: null # precedence over max_epochs
+    accumulate_grad_batches: 1 # accumulates grads every k batches
+    amp_level: O0 # O1/O2 for mixed precision
+    distributed_backend: ddp
+
+model:
```

Comment:
The entire model level is passed as a whole into the PTL model that we are initializing. Think of it as basically the super __init__ of the entire PTL model.

Reply:
Model corresponds to NeMo's abstraction of model:

>Definition. A NeMo model is an entity which contains model (e.g. Neural Network plus all necessary pre and post-processing), datasets logic and optimizer together with scheduler. It is an instance of LightningModule and can be used as such. Because it is an instance of torch.nn.Module as well it should be usable as such too. Finally, it contains input and output types and it's forward method is typed.
>
>Inherit from nemo.core.ModelPT. Note, that this will bring pytorch_lightning.LightningModule inheritance.

This way we can configure a NeMo Model with a single config object. For text classification it's TextClassificationModelConfig

Reply:
yes, we should have model and pl levels. But I'm not so sure about "io" level